crates/sui-indexer/tests/integration_tests.rs

```diff
@@ -438,30 +436,6 @@ pub mod pg_integration_test {
         assert_eq!(digest_one, query_response.data[0].id.tx_digest);
         assert_eq!(digest_two, query_response.data[1].id.tx_digest);
 
-        // Verify that the transfer coin event occurred successfully, without emitting an event
```

Comment:
this test is not very useful we know for sure that transferring object does not emit an event